YoCode/PrintToConsole.cs

```diff
@@ -60,6 +60,13 @@ public void PrintError(List<string> errs)
             Console.WriteLine("\nIf you would like to see list of commands, type: --help");
         }
 
+        public void PrintHelp()
+        {
+            Console.WriteLine($"Application takes 2 parameters: path to original test directory and path to modified test directory" +
```

Comment:
You don't need the `$` on that string because it doesn't contain any interpolation. Not a big deal though.